.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - '4'
+  - '6'
+script:
+  - npm run coverage
+  - npm run codecov
+notifications:
+  email:
+    on_success: change
+    on_failure: change
```

Comment:
It works, but I was surprised it does. Really don't like getting emails about it though so Hipchat notifications sound good. I can't really figure out how to make it work though, @hisabimbola could you take a look at this?